src/toolbox.js

```diff
@@ -161,6 +170,15 @@ var Toolbox = GObject.registerClass({
         topic.get_style_context().remove_class('reveal');
     }
 
+    selectTopic(id) {
+        const [topicRow, topic] = this._findTopic(id); // eslint-disable-line
+        // Set milisecs in timeout to length of  lockscreen video
```

Comment:
There appears to be a `done` signal on the playbin widget which it looks like you could connect to instead of hardcoding the video length. I'm not sure exactly how that works, but Juan Pablo would have more details if you need them.

Nice touch, by the way, to select the topic once the lockscreen opens.